_posts/2015-02-09-create-your-application.markdown

```diff
@@ -71,19 +71,19 @@ For more information about generating a Rails application see the [Getting Start
     gem 'jettywrapper'
     ```
 
-  1. Install gem and dependencies
+  1. Install required gems and their dependencies
 
     ```sh
     $ bundle install
     ```
 
-  1. Run Blacklight generator
+  1. Run Blacklight generator (with devise authentication)
 
     ```sh
     $ rails g blacklight:install --devise
     ```
 
-  1. Run GeoBlacklight generator
+  1. Run GeoBlacklight generator (with a jetty instance for Solr)
```

Comment:
This is actually just configuring jetty appropriately for GeoBlacklight rather than Hydra. Maybe it could be 

1. Run GeoBlacklight generator (with jetty version configuration for GeoBlacklight)

?